### ISSUE TITLE
"module 'salt.ext' has no attribute 'ipaddress'" on using network.convert_cidr

### ISSUE THREAD

**Issue by JensErat** [saltstack/salt#49588]
### Description of Issue/Question

When running the `network.convert_cidr` example command from the documentation, a python stack trace is thrown.

It seems the [issue is already fixed (without the author realizing there was an issue)](https://github.com/saltstack/salt/commit/24669f79f244bb951c0eec7d958171c6268cfcbf#diff-670a7e2e3a706378b2191ed8b28f4ccf): but the fix is neither backported to [2017.7.8](https://github.com/saltstack/salt/blob/2017.7.8/salt/modules/network.py#L1138) nor [2018.3.3](https://github.com/saltstack/salt/blob/2018.3.3/salt/modules/network.py#L1143).

### Setup

Salt installed on Ubuntu 18.04 from Ubuntu repositories.

### Steps to Reproduce Issue

```bash
# salt '*' network.convert_cidr 172.31.0.0/16
STR-01-RTR-001:
    The minion function caused an exception: Traceback (most recent call last):
      File "/usr/lib/python3/dist-packages/salt/minion.py", line 1493, in _thread_return
        return_data = executor.execute()
      File "/usr/lib/python3/dist-packages/salt/executors/direct_call.py", line 28, in execute
        return self.func(*self.args, **self.kwargs)
      File "/usr/lib/python3/dist-packages/salt/modules/network.py", line 1138, in convert_cidr
        network_info = salt.ext.ipaddress.ip_network(cidr)
    AttributeError: module 'salt.ext' has no attribute 'ipaddress'
```

### Versions Report

```
# salt --versions-report
Salt Version:
           Salt: 2017.7.4
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.6.1
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.10
        libgit2: 0.26.0
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.5.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: 0.26.2
         Python: 3.6.5 (default, Apr  1 2018, 05:46:30)
   python-gnupg: Not Installed
         PyYAML: 3.12
          PyZMQ: 16.0.2
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: 4.2.5
 
System Versions:
           dist: Ubuntu 18.04 bionic
         locale: ANSI_X3.4-1968
        machine: x86_64
        release: 4.15.0-33-generic
         system: Linux
        version: Ubuntu 18.04 bionic
```

**Comment by garethgreenaway** [comment]
@JensErat Thanks for the report.  I've marked this issue as well as the PR in question to be back ported to 2018.3 and 2017.7.

**Comment by rallytime** [comment]
Hi @JensErat - I have back-ported that fix to the 2017.7 branch in PR #49597 (the fix will then be merge-forward to 2018.3 as well.)

Thanks for letting us know!
